server.go

```diff
@@ -475,16 +491,20 @@ func (s *server) handleQuery(querymsg interface{}, state *peerState) {
 		msg.reply <- infos
 
 	case addNodeMsg:
+	case connectNodeMsg:
```

Comment:
Yeah you're right. Trying to use the `addnode` sub-commands, `del` or `add` causes the `btcctl` caller to hang indefinitely waiting for a reply since the case stacking essentially made the commands into a no-op. 

I agree that we should mark `addnode` as deprecated and point users to the new `node` command instead. But for now, 've created as a temporary PR fix: btcsuite/btcd#398